Packs/BmcHelixRemedyForce/Integrations/BmcHelixRemedyForce/BMCHelixRemedyforce.py

```diff
@@ -0,0 +1,2883 @@
+    :param excluded_fields: List of the fields which will not updated in service request.
+
+    :type additional_fields: ``dict``
+    :param additional_fields: Dictionary containing key-value pairs which will be passed as "additional_fields"
+                              argument.
+
+    :type default_args: ``dict``
+    :param default_args: Dictionary containing key-value pairs of default arguments of command.
+
+    :raises DemistoException: If any issues will occur while making the http call to create service request.
+    """
+    category_id = default_args.get("category_id")
+    queue_id = default_args.get("queue_id")
+    staff_id = default_args.get("staff_id")
+    status_id = default_args.get("status_id")
+    urgency_id = default_args.get("urgency_id")
+    client_id = default_args.get("client_id")
+    try:
+        body = {
+            "Fields": [
+                {
+                    "Name": "requestDefinitionId",
+                    "Value": service_request_definition
+                },
+                {
+                    "Name": "client",
+                    "Value": client_id if client_id else ""
+                }
+            ],
+            "Answers": answers
+        }
+        headers = {
+            "Content-Type": "application/json"
+        }
+        response = client.http_request(method='POST', url_suffix=URL_SUFFIX["SERVICE_REQUEST"], headers=headers,
+                                       json_data=body)
+    except DemistoException as e:
+        raise DemistoException(str(e) if str(e) else MESSAGES['UNEXPECTED_ERROR'])
+    if response and response.get("Success"):
+        outputs = {
+            "Number": response.get('Result', {}).get('Number', 0),
+            "Id": response.get('Result', {}).get('Id', 0),
+            "CreatedDate": datetime.now().strftime(DATE_FORMAT)
+        }
+        markdown_message = "{}".format(HR_MESSAGES["SERVICE_REQUEST_CREATE_SUCCESS"]).format(
+            response.get('Result', {}).get('Number', 0))
+        params = {
+            "category_id": category_id,
+            "queue_id": queue_id,
+            "staff_id": staff_id,
+            "status_id": status_id,
+            "urgency_id": urgency_id,
+            "client_id": client_id
+        }
+        if additional_fields:
+            params = get_request_params(data=additional_fields, params=params)
+        params = remove_empty_elements(params)
+        resp = update_incident(client, response.get('Result', {}).get('Id', 0), params=params)
+        if resp.get("message"):
+            markdown_message = "{}".format(MESSAGES["CREATE_SERVICE_REQUEST_WARNING"]).format(
+                response.get('Result', {}).get('Number', 0), ", ".join(params.keys()), resp.get("message"))
+            hr_output = {
+                OUTPUT_PREFIX['SERVICE_REQUEST_WARNING']: outputs
+            }
+            demisto.error(traceback.format_exc())  # print the traceback
+            return_warning(
```

Comment:
@liorblob , we are returning warning here and `return_warning` function does not handle dt key part. So, we have written in `OUTPUT_PREFIX` for that only. For others we are using `CommandResult`.